llvm/include/llvm/IR/RepoDefinition.h

```diff
@@ -169,17 +179,14 @@ class ModuleHashGenerator {
   template <typename GlobalType> void calculateGOInfo(const GlobalType *G);
   void calculateGOInfo(const GlobalObject *GO);
 
-  /// Accumulate the GO's digest and get its GODigestState.
+  /// Accumulate the GO's digest and get its Dependencies.
```

Comment:
So I understand, we can just use Dependencies. Is that why the GODigestState class is no longer required?

Reply:
Yes, it is.